tests/classes/search/ArticleSearchIndexTest.php

```diff
@@ -177,15 +177,6 @@ public function testIndexArticleMetadata()
         // Make sure that no hook is being called.
         HookRegistry::clear('ArticleSearchIndex::articleMetadataChanged');
 
-        // Mock an article so that the authors are not
```

Comment:
Rather than removing this code entirely, I'd recommend leaving it in place and adding a `FIXME` noting that it hasn't been adapted for this change.

Reply:
(cross-linking https://github.com/pkp/ops/pull/172/files#r692736180)

Reply:
Fixed